$OpenBSD: patch-src_nautilus-canvas-container_c,v 1.5 2017/07/18 07:27:38 ajacoutot Exp $

nautilus-canvas-container.c:5783: error: #pragma GCC diagnostic not allowed inside functions
nautilus-canvas-container.c:5784: error: #pragma GCC diagnostic not allowed inside functions
nautilus-canvas-container.c:5792: error: #pragma GCC diagnostic not allowed inside functions

Index: src/nautilus-canvas-container.c
--- src/nautilus-canvas-container.c.orig
+++ src/nautilus-canvas-container.c
@@ -5813,8 +5813,6 @@ get_text_ellipsis_limit_for_zoom (char       **strs,
 
     if (strs != NULL)
     {
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wformat-nonliteral"
         for (p = strs; *p != NULL; p++)
         {
             if (sscanf (*p, str, limit))
@@ -5822,7 +5820,6 @@ get_text_ellipsis_limit_for_zoom (char       **strs,
                 success = TRUE;
             }
         }
-#pragma GCC diagnostic pop
     }
 
     g_free (str);
